plugins/inputs/nomad/nomad.go

```diff
@@ -0,0 +1,215 @@
+package nomad
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Nomad struct {
+	URL string
+
+	AuthToken       string `toml:"auth_token"`
+	AuthTokenString string `toml:"auth_token_string"`
+
+	LabelInclude []string `toml:"label_include"`
+	LabelExclude []string `toml:"label_exclude"`
+
+	labelFilter filter.Filter
+
+	ResponseTimeout config.Duration
```

Comment:
Please also add the corresponding TOML annotation. Furthermore, it improves the overview if you group (exported) user options and internal member variables.